### ISSUE TITLE
Bug with @types/package dependencies

### ISSUE THREAD

**Issue by lholznagel** [yarnpkg/yarn#651]
**Do you want to request a *feature* or report a *bug*?**
Possible bug found

**What is the current behavior?**
When the `package.json` contains dependencies like `@types/node` it fails on `yarn outdated` with the following error.

Error:
```
error https://registry.yarnpkg.com/@types/node: Not found
    at Request.params.callback [as _callback] (/usr/share/yarn/lib-legacy/util/request-manager.js:285:18)
    at Request.self.callback (/usr/share/yarn/node_modules/request/request.js:187:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (/usr/share/yarn/node_modules/request/request.js:1048:10)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (/usr/share/yarn/node_modules/request/request.js:969:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
```

**If the current behavior is a bug, please provide the steps to reproduce.**
1. Install `@types/node` (`yarn add @types/node`)
2. Run `yarn outdated` or `yarn outdated @types/node`
3. The error should occur

**What is the expected behavior?**
It should check if `@types/node` is outdated and should not throw an error

**Please mention your node.js, yarn and operating system version.**
node.js: `6.7.0`
yarn: `v0.15.0`
os: `Ubuntu 16.04`

**Comment by ssreekanth** [comment]
Duplicate of #620

**Comment by lholznagel** [comment]
Oh sorry, didn´t find that while searching.

This then can be closed in favor of #620
